composer.json

```diff
@@ -10,8 +10,8 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": "^7.2.0|^8.0",
-        "symfony/cache": "^3.4|^4.4|^5.0",
+        "php": "^7.2.0|^8.0|^8.1",
```

Comment:
I was wondering why you added this, are you have a specific issue? `^8.0` would already include `8.1.x`

Reply:
My bad, I removed the PHP bump from the PR. I was having issues using the client due to using Symfony 6, which was incompatible with the symfony/cache version provided by the client.